lib/models/navigation/page_data/problem_page_data.dart

```diff
@@ -0,0 +1,31 @@
+library problem_page_data;
+
+import 'dart:convert';
+
+import 'package:built_value/built_value.dart';
+import 'package:built_value/serializer.dart';
+import 'package:crowdleague/models/navigation/page_data/page_data.dart';
+import 'package:crowdleague/models/problems/problem_base.dart';
+import 'package:crowdleague/utils/serializers.dart';
+import 'package:meta/meta.dart';
+
+part 'problem_page_data.g.dart';
+
+abstract class ProblemPageData
```

Comment:
Would you be up for putting some clothes on this guy? :-)

Reply:
Yep! I almost just pushed another commit to do just that. Workin'!

Reply:
Looks great!